src/modules/integrator-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -2,74 +2,124 @@ include::ROOT:partial$variables.adoc[]
+The data privacy console enables you to access your data.
+It provides a platform where you may submit requests to modify, delete, and export your data.
+These requests (deletion, correction, and export) concerning the treatment of your data are forwarded to the xref:data-privacy/data-privacy-officer.adoc[data privacy officer (DPO)].
 
 This feature is also enabled to allow all users to browse locations where the data is stored.
 
 .The screenshot shows the overview of the data privacy console.
 image::data-privacy/data-privacy-console.png[role="data-zoomable"]
 
-== Features of the Privacy Console
+// == Data Overview
 
-* Data Overview
-* Privacy Requests
+// Data overview allows you to review your profile information and your data stored in Corteza applications.
 
-== Data Overview
+// image::data-privacy/data-overview.png[role="data-zoomable"]
 
-Data overview allows you to review your profile information and your personal data stored in Corteza applications.
+// * Profile Information - Contact information, names, surnames, nicknames, profile picture.
 
-image::data-privacy/data-overview.png[role="data-zoomable"]
-
-* Profile Information - Contact information, names, surnames, nickname, profile picture.
-
-* Data Application - Personal data stored in your {PRODUCT_NAME} instance.
+// * Data Application - Personal data stored in your {PRODUCT_NAME} instance.
 
 == Privacy Requests
 Access, manage or export a list of all your privacy-related requests.
 
-=== Types of privacy requests
-
 [cols="1s,5a"]
 |===
-| [#privacy-request-correction]#<<Correction-Request,Correction Request>>#
+| [#correction-Request]#<<correction-Request,Correction Request>>#
 |
-When a user is unsatisfied with their personal data because it may be inaccurate, incomplete, misleading or not up-to-date, they can request to correct or delete the particular data.
-
-To make a correction request:
-
-. Navigate to menu:privacy[Data Privacy Console]
-. Click on *[Data Overview]* 
-. Select Application data(Your data is stored in corteza).
-. Click on the connection dropdown to select the type of connection
-. Fill in the data, i.e. with the corrections.
-. Click the *[Request Correction]* button.
-
-
-| [#privacy-request-deletion]#<<Deletion-request,Deletion request>>#
+When a user is dissatisfied with their personal information because it is inaccurate, incomplete, misleading, or out-of-date, they can request that the information be rectified or deleted.
+
+To request a correction:
+
+. Navigate to the Data Privacy Console by clicking on btn:[Privacy]
+. Click on btn:[Data Overview] 
+. Select Application data(Your data is stored in {PRODUCT_NAME} instance).
+. Select the type of connection by selecting it from the Connection dropdown. 
+. Fill in the data with the necessary corrections. 
+. Click the btn:[Request Correction] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "data-privacy/request-correction.png",
```

Comment:
Please replace this screenshot with one that shows some data instead of the empty state.
I have some setup locally so if you need I can help you get some screenshots as well.